analysis/paper/paper.Rmd

```diff
@@ -31,7 +31,8 @@ knitr::opts_chunk$set(
   fig.path = "figures"
 )
 
-library(rrtools)
+# Load the current package with all its environment specific objects 
+devtools::load_all(".", quiet = T) # you may replace this with library(package_name)
```

Comment:
I'm not sure if I understand this correctly, but I think `library(rrtools)` was wrong in the first place. It should have been `library(package_name)`. Therefore I think `devtools::load_all(".", quiet = T)` is good solution. On the other hand @benmarwick is also right: This is a matter of taste. I personally would remove this line and call all the package functions and data manually with the namespace declaration (`package_name::my_cool_function`) just to make it very clear, where something is coming from.

Reply:
Yes, in fact, when we generate the paper.Rmd in our new compendium, using `rrtools::use_analysis()`, we do get `library(package_name)`. It's only in this demonstration file that we see `library(rrtools)`, because this particular pkg is also called rrtools.